src/viam/sdk/robot/client.cpp

```diff
@@ -116,7 +117,9 @@ std::vector<Operation> RobotClient::get_operations() {
     if (is_error_response(response)) {
         BOOST_LOG_TRIVIAL(error) << "Error getting operations: " << response.error_message();
     }
+
     for (int i = 0; i < resp.operations().size(); ++i) {
+        // NOLINTNEXTLINE
         operations.push_back(resp.operations().at(i));
```

Comment:
Strangely, creating `operations` with `std::vector<Operation> operations(resp.operations().size())` ends up with the wrong size vector. I have not investigated further than that.